sorc/ncep_post.fd/CALHEL.f

```diff
@@ -444,6 +444,10 @@ SUBROUTINE CALHEL(DEPTH,UST,VST,HELI,USHR1,VSHR1,USHR6,VSHR6)
                 DU2 = UH(I,J,L)-UH(I,J,L-1)
                 DV1 = VH(I,J,L+1)-VH(I,J,L)
                 DV2 = VH(I,J,L)-VH(I,J,L-1)
+               IF( VH(I,J,L)  <spval.and.UH(I,J,L)  <spval.and.         &
+                   VH(I,J,L+1)<spval.and.UH(I,J,L+1)<spval.and.         &
+                   VH(I,J,L-1)<spval.and.UH(I,J,L-1)<spval.and.         &
+                   VST(I,J)   <spval)                                   &
```

Comment:
@JesseMeng-NOAA add checking UST(I,J) in line 450?

Reply:
will do

Reply:
done